opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectBookmark.jsx

```diff
@@ -146,14 +148,20 @@ class StixDomainObjectBookmarkComponent extends Component {
         >
           <CardHeader
             classes={{ root: classes.header }}
-            avatar={
+            avatar={ node.images && node.images.length > 0 ? (
```

Comment:
This is a generic component, so we have the mechanism in Threat Actor group, Instrusion sets ... bookmark as well, but not in all the card of this entities.
Example : 
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/32be7d9a-71fb-48e8-9ff9-d50a565c3a76)